--- a/.env.template	2020-10-11 11:11:30.287560964 +0200
+++ b/.env.template	2020-10-11 11:12:42.830465654 +0200
@@ -5,7 +5,7 @@
 ## in the admin interface. Those overrides are stored within DATA_FOLDER/config.json .
 
 ## Main data folder
-# DATA_FOLDER=data
+DATA_FOLDER=/usr/share/bitwarden-rs/data
 
 ## Database URL
 ## When using SQLite, this is the path to the DB file, default to %DATA_FOLDER%/db.sqlite3
@@ -45,8 +45,8 @@
 # ICON_CACHE_NEGTTL=259200
 
 ## Web vault settings
-# WEB_VAULT_FOLDER=web-vault/
-# WEB_VAULT_ENABLED=true
+WEB_VAULT_FOLDER=/usr/share/bitwarden-rs/web-vault
+WEB_VAULT_ENABLED=true
 
 ## Enables websocket notifications
 # WEBSOCKET_ENABLED=false
